hack/windows/Helper.psm1

```diff
@@ -245,7 +245,6 @@ function Start-OVSServices {
 function Start-AntreaAgent {
     Param(
         [parameter(Mandatory = $false, HelpMessage="Antrea home path")] [string] $AntreaHome="c:\k\antrea",
-        [parameter(Mandatory = $false, HelpMessage="kubeconfig file path")] [string] $KubeConfig="c:\k\config",
```

Comment:
The parameter is still used by the code in the script.

Reply:
But I didn't find this KubeConfig being used in Start-AntreaAgent.

Reply:
@tnqn Hi, do you have more comments?